cluster_tolerance: 0.15
min_cluster_size: 35
max_cluster_size: 15000
filter_square_diameter: 0.5
